modules/ossm-cert-manager-install-istio-resource.adoc

```diff
@@ -0,0 +1,149 @@
+// Module included in the following assemblies:
+//
+// * service-mesh-docs-main/install/ossm-cert-manager-assembly.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="install-istio-resource_{context}"]
+= Install your Istio resource
+
+//TP1 content influx. Title, etc may change.
+//Content is very similar to 2.x content
+//all kinds of formatting things to fix. want to see if a build will generate to have a look, and see how it fits structurally with the IA.
+
+
+After you have installed `istio-csr` by following the procedure for either `InPlace` or `RevisionBased`, you can install the `Istio` resource.
+
+This disables Istio's built in CA server and points istiod to the `istio-csr` CA server. The `istio-csr` CA server issues certificates for both istiod and user workloads. Additionally, the istiod TLS cert is mounted into the pod at a known location where it will be read. is mounted in a known location where it will be read by istiod. Mounting the certificates to a known location is only necessary for {SMProduct}.
+
+.Procedure
+
+. Create the `Istio` object as in the following example:
++
+.Example `Istio` object
```

Comment:
Should this be `Example istio.yaml` to be consistent with the `issuer.yaml` example and since we do `oc apply -f istio.yaml` next?